can_tools/scrapers/base.py

```diff
@@ -140,4 +155,7 @@ def put(self, connstr: str, df=None):
             msg = "field `pk` must be set on subclass of OnConflictNothingBase"
             raise ValueError(msg)
 
+        if self.data_type == "covid":
```

Comment:
Do we want to use the `_ensure_covid_demograhpics` method?

I continue to oppose the idea of inserting uninspected meta information into the database, but I'm open to some push back here.

Reply:
Nope, we shouldn't use it